### ISSUE TITLE
[FEATURE] Better naming conventions

### ISSUE THREAD

**Issue by Bia10** [OSRSB/OsrsBot#111]
**Is your feature request related to a problem? Please describe.**

The class named GlobalSettingValues hardly relates to what it contains.

The class contains mostly offsets to PlayerVars not the Vars themselves neither the actual values of the Vars.

GlobalSettingValues makes it sound like the actual values of Vars was contained there. Nope that we first have .getVar(id) then .getValue() to get the actual value.

- [X]  GlobalSettingValues -> VarpIndices
- [X] getSettingArray() -> getLoadedVarpValues()
- [X] getSetting(final int setting) -> getVarpValueAt(final int index)
- [X] Settings -> ClientLocalStorage

Edit: Although this has been addressed in #119 ,  keep this open as i have not browsed more then a few files, i am sure i will become frustrated with naming as i browse more source so this issue will last a while.

The name of package  _/internal/naturalmouse_ is non-sensical as have explain on discord.  There is no need for any distinction between a natural and un-natural mouse, from point of jagex server its a distinction between patterns left by a human-like mouse and machine-like mouse. If you actually create a robotic hand to learn to perform identical mouse movements and key presses like a human does its a human-like mouse from point of jagex servers even if its entirely un-natural as being entirely artificial.

**Comment by GigiaJ** [comment]
Yeah, I realize now that the name for the offsets is ambiguous as we've mentioned. I'll go ahead and re-name it shortly.
If I'm honest I didn't name the natural mouse library. I know we've spoken about it.

**Comment by GigiaJ** [comment]
Those names are far better than what is currently in place. I'm all for it.
